py/src/ai/h2o/sparkling/VersionComponents.py

```diff
@@ -20,13 +20,23 @@
 class VersionComponents(object):
 
     @staticmethod
-    def parseFromVersion(version):
+    def parseFromSparklingWaterVersion(version):
         match = re.search(r"^((\d+\.\d+).(\d+))(-(\d+))?-(\d+.\d+)$", version)
         result = VersionComponents()
         result.fullVersion = match.group(0)
         result.sparklingVersion = match.group(1)
         result.sparklingMajorVersion = match.group(2)
         result.sparklingMinorVersion = match.group(3)
         result.nightlyVersion = match.group(5)
-        result.sparkVersion = match.group(6)
+        result.sparkMajorVersion = match.group(6)
```

Comment:
nit: Good point with `sparkVersion`, it's not the full version! But since Spark uses versioning schame `major.minor.patch` would other people associate it with `2` instead of `2.4`?

Reply:
We can call it as `result.sparkMajorMinorVersion`?

However we don't expose( I would say on purpose) the VersionComponents from `ai.h2o package` into `pysparkling` which is main entry point to PySparkling users.

But agree, will change it so are definitely sure what it means 👍 Thanks!

Reply:
`sparkMajorMinorVersion` sounds good to me ;-)